index.bs

```diff
@@ -2043,18 +2044,32 @@ is a request, rather than a guarantee.
 
 <h3 id="string-constants">Use strings for constants and enums</h3>
 
-If your API needs a constant, or a set of enumerated values,
-use string values.
+Use strings as the values for constants or sets of enumerated values.
 
-Strings are easier for developers to inspect,
-and in JavaScript engines there is no performance benefit
-from using integers instead of strings.
+Strings are easier for developers to inspect values
+and read code that uses those values.
+In JavaScript engines there is no performance benefit
+to using integers instead of strings.
+The [WebIDL enum type](https://webidl.spec.whatwg.org/#idl-enums)
+only permits the use of string values.
 
 If you need to express a state which is a combination of properties,
 which might be expressed as a bitmask in another language,
 use a dictionary object instead.
 This object can be passed around as easily as a single bitmask value.
 
+<div class=example>
```

Comment:
Add `id`s to the examples, to make deep links more stable.

Reply:
I'm going to suggest that we do that systemically, rather than start here.  (I am not sure that examples need that sort of link stability, but if we want that sort of stability, then we should do it properly.)